watchdog.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import schedule
+import time
+import node_monitor.load_config as c
+from typing import List
+from node_monitor.bot_email import EmailBot
+
+CHECK_INTERVAL = 15*60
+
+class Watchdog:
+    def __init__(
+            self, 
+            email_bot: EmailBot, 
+            recipients: List[str],
+            node_monitor_url: str) -> None:
+        self.email_bot = email_bot
+        self.recipients = recipients
+        self.node_monitor_url = node_monitor_url
+        self.jobs = [
+            schedule.every(15).minutes.do(self.check_node_monitor_status)
+        ]
```

Comment:
You don't need to use `schedule` at all actually.

You can just run everything in a `mainloop` function and just sleep for `CHECK_INTERVAL_SECONDS`